public/schema/JobTemplates-01.yaml

```
@@ -28,7 +28,7 @@ properties:
                 description: A test suite with machine and/ or priority specified, or a custom job template name if testsuite was specified
                 additionalProperties: false
                 patternProperties:
-                  "^[A-Za-z\\s0-9_*+-]+$":
+                  "^[A-Za-z\\s0-9_*+-.]+$":
```

```suggestion
                  "^[A-Za-z\\s0-9_*.+-]+$":
```

Your version: "+-. a single character in the range between + (index 43) and . (index 46) (case sensitive)"
My version: "_*.+- matches a single character in the list _*.+- (case sensitive)"